apps/emqx/src/emqx_persistent_session_ds/emqx_persistent_session_ds_subs.erl

```diff
@@ -74,27 +85,17 @@
         _ => _
     },
     %% Optional field that is added when subscription state becomes
-    %% outdated:
-    superseded_by => subscription_state_id()
+    %% outdated (note: do not use the value, as GC may delete
+    %% subscription states referenced by `superseded_by'):
+    superseded_by => subscription_state_id(),
+    %% Optional field used by shared subscriptions:
+    share_topic_filter => #share{}
```

Comment:
As far as I see, it is not used now actually. I'll make a task for myself to consistently remove this field after merge.